adafruit_l3gd20.py

```diff
@@ -346,7 +358,7 @@ def read_register(self, register):
             # print("$%02X => %s" % (register, [hex(i) for i in self._spi_bytearray1]))
             return self._spi_bytearray1[0]
 
-    def read_bytes(self, register, buffer):
+    def read_bytes(self, register: int, buffer: bytearray) -> None:
```

Comment:
Let's change this to `circuitpython_typing.WriteableBuffer`, to make it more generalized.